metadata/v1.23.0/proposal_1_upgrade_framework.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Multi-step proposal to upgrade mainnet framework, version 1.23.0",
+  "description": "This includes changes in https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.23.2",
+  "source_code_url": "https://github.com/aptos-labs/aptos-core",
```

Comment:
source code url need to be the move files in this repo

Reply:
Done.